extensions/common/aws/aws-s3-test/src/testFixtures/java/org/eclipse/edc/aws/s3/testfixtures/AbstractS3Test.java

```diff
@@ -68,12 +68,14 @@ public abstract class AbstractS3Test {
     // [see http://stackoverflow.com/questions/13898057/aws-error-message-a-conflicting-conditional-operation-is-currently-in-progress]
     protected static final String MINIO_ENDPOINT = "http://localhost:9000";
     protected static final URI S3_ENDPOINT = URI.create(propOrEnv("it.aws.endpoint", MINIO_ENDPOINT));
+    protected static final Boolean PATH_STYLE_ACCESS_ENABLED = Boolean.valueOf(propOrEnv("it.aws.path.style.access.enabled", "true"));
```

Comment:
this propOrEnv is not set by any test, I wold hardcode the `true` in the configuration.

Reply:
This is for manual testing with endpoint override. We can run test against real S3 with virtual-hosted style addressing
by following command line. Same for another S3 compatible storage supporting.

```
$ IT_AWS_ENDPOINT=https://s3.ap-northeast-1.amazonaws.com/ \
  IT_AWS_REGION=ap-northeast-1 \
  IT_AWS_PROFILE=iwasakims \
  IT_AWS_PATH_STYLE_ACCESS_ENABLED=false \
    ./gradlew clean test -p extensions/control-plane/provision/provision-aws-s3 -DincludeTags=AwsS3IntegrationTest -PverboseTest
```

Reply:
if so, please document it within the `README` file of the module